core/domain/exp_services.py

```diff
@@ -786,7 +786,7 @@ def delete_explorations(committer_id, exploration_ids, force_deletion=False):
 
 
 def delete_explorations_from_subscribed_users(exploration_ids):
-    """Remove explorations from all subscribers' activity_ids.
+    """Remove explorations from all subscribers' entity_ids.
```

Comment:
Yes, mistake.

Reply:
Yes, mistake.

Reply:
Er ... not fixed? Or the docstring should be modified somehow, because "entity_ids" is too vague in this context...

Please double-check elsewhere that all the terms are being used correctly, too.